sig/shims/open-uri.rbs

```diff
@@ -0,0 +1,4 @@
+module URI
```

Comment:
Note: This type definition has been merged into the upstream ([ruby/rbs](https://github.com/ruby/rbs/pull/1592)). Therefore this file will be unneeded after rbs-3.3.0 release. But this is needed at this moment...